test/testRunner.c

```diff
@@ -0,0 +1,11 @@
+#include "ventilation-test.h"
+#include "testUtil.h"
+
+int main(int argc, char **argv) {
+  //test(addition, "addition");
```

Comment:
remove unused test? Or are you going to commit this later?

Reply:
removed